### PR TITLE
Feat/notification system - Team Bulldozer

### PR DESCRIPTION
**Pull request by gh-i19n**
# Description

**Closes #issue_number_here**

# Changes proposed

## What were you told to do?

Implement the notification settings system

## What did you do?
I implemented the design of the notification system, made it mobile reponsive and also set up a workflow for api integration, which will come in shortly.

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

Screenshot 2024-07-30 144954
Screenshot 2024-07-30 144930
Screenshot 2024-07-30 144916
Screenshot 2024-07-30 145008

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -166,7 +166,7 @@
 /* Custom Scrollbar Styling */
 /* For Webkit Browsers */
 ::-webkit-scrollbar {
-  width: 14px; /* Width of the scrollbar */
```
Comment (prudentbird) [inline src/app/globals.css:169]: Is this change necessary????????????????????????????

I would overlook though

**Review by prudentbird — APPROVED** [review]

File: src/app/globals.css (hunk 1 above)
Comment (gh-i19n) [reply src/app/globals.css:169]: yh, the scrollbar is taking too much real estate on the mobile view, reducing the bar width by 2px solves the issue.
